pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -211,8 +211,8 @@ Bento supported AMP components are listed below:
 - [amp-accordion](../../../documentation/components/reference/amp-accordion-v1.0.md)
 - [amp-base-carousel](../../../documentation/components/reference/amp-base-carousel-v1.0.md)
 - [amp-inline-gallery](../../../documentation/components/reference/amp-inline-gallery-v1.0.md)
-  - amp-inline-gallery-pagination
-  - amp-inline-gallery-thumbnails
+    - amp-inline-gallery-pagination
```

Comment:
Didn't realize these would look funny without links themselves. Thoughts on including link to `#include-pagination-indicators` and `#include-thumbnails`?

Reply:
I've added links :)